CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
@dsogari I believe it is possible to re-open a PR after a force push updates the branch for it. I think the original PR's author has to reopen it. The reopen button is grayed out for me.